pom.xml

```diff
@@ -22,6 +22,7 @@
     <module>beekeeper-integration-tests</module>
     <module>beekeeper-package</module>
     <module>beekeeper-vacuum-tool</module>
+    <module>beekeeper-metadata-cleanup</module>
```

Comment:
module should probably go above the integration tests, i think order here affects build order

Reply:
ah okay, good point. I will move this